transformer/app.js

```diff
@@ -195,9 +195,12 @@ export async function lambdaHandler(event, context) {
   logger.debug('Lambda triggered by SQS message', { event });
   try {
     const accountId = process.env.USE_DEV_ACCOUNT_ID === 'true' ? process.env.DEV_ACCOUNT_ID : context.invokedFunctionArn.split(':')[4];
+    const xrayArn = process.env.USE_DEV_ACCOUNT_ID === 'true'
```

Comment:
Let's add `.toLowerCase()` here to ensure we are comparing correctly. Not a blocker though.